spec/shared_examples/user_reset_password_shared_examples.rb

```diff
@@ -314,17 +316,27 @@
       end
     end
 
-    it 'when change_password! is called, deletes reset_password_token' do
+    it 'when change_password is called, deletes reset_password_token' do
       user.deliver_reset_password_instructions!
 
       expect(user.reset_password_token).not_to be_nil
 
-      user.change_password!('blabulsdf')
-      user.save!
+      user.change_password('blabulsdf')
 
       expect(user.reset_password_token).to be_nil
     end
 
+    it 'when change_password! is called, calls change_password' do
```

Comment:
This test seems a bit out of place. Testing that both the normal and bang method work makes sense, but testing that one calls the other doesn't seem beneficial.